### DISCUSSION TITLE
Webview2 Manager in OrcaSlicer Slow Upload to Printer

### DISCUSSION THREAD

**Discussion by d3dorsett** [OrcaSlicer/OrcaSlicer#7515] [Q&A]
I am using OrcaSlicer 2.2.0 and I have noticed that when I upload a sliced STL to my K1 Max, it is very slow if I have been using OrcaSlicer for about 15-30 minutes. If I exit OrcaSlicer and open it again, I can quickly upload an STL.  I have noticed that Webview2 Manager seems to be taking up more resources that I think it should. Watching it in task manager on Windows 11, the memory usage continues to grow and grow. Eventually it will time out. Has anyone else noticed this issue? Any suggestionb?

![image](https://github.com/user-attachments/assets/a9f0cc7c-79d3-4998-8460-361a5aac22b0)

**Comment by LJKraus** [comment]
I've not seen that problem, but Webview 2 is a Microsoft product.  You might try uninstalling and reinstalling from Windows Settings>Apps to update/repair the installation.

**Comment by gregazar** [comment]
Get rid of Webview2, it's a serious privacy issue and intellectual property issue. Ask Ai, (not Copilot) but don't ask Ai unless you are prepared to ditch WebView2 and Edge, you won't like the answer.

I removed MS Edge and all related components and blocked them from returning. Including Copilot, Start Experiences and about a dozen other data harvesting tools that MS has slipped in on ALL users starting in late 2025. Another Ai question you won't like the answer to.

I'm telling you, MS is just as evil and is it's pedo-founder. They are gathering data on you and they don't deny it. They know most people have no choice but use Windows.
